src/env/dev/terraform.tfvars

```
@@ -232,3 +233,12 @@ buyerbanks_function_autoscale_minimum = 1
 buyerbanks_function_autoscale_maximum = 3
 buyerbanks_function_autoscale_default = 1
 buyerbanks_delete_retention_days      = 30
+
+# pagopa-proxy app service
+pagopa_proxy_enabled                        = true
+pagopa_proxy_redis_capacity                 = 1
```

```suggestion
pagopa_proxy_redis_capacity                 = 0
```
zero is the min value